Configuration/NodeTypes.yaml

```diff
@@ -19,11 +19,11 @@
         stats:
           label: 'Neos.GoogleAnalytics:NodeTypes.StatsTabMixin:tabs.stats'
           position: 100
-          icon: 'icon-line-chart icon-chart-line'
+          icon: 'chart-line'
```

Comment:
this breaks backwards compatibility.. just add chart-line in addition to the other ones so we don't need a new version for the latest version

Reply:
Hmm, wenn I simply add '`chart-line` to the other ones no Icon is shown at least in the new UI:
`icon: 'icon-line-chart icon-chart-line chart-line'`

Reply:
did you try adding it as the first instead? if that doesn't work I guess they're changed how the selector works as well